### ISSUE TITLE
BUG: Editing Content through modules loaded through the skin saves changes but throws an error

### ISSUE THREAD

**Issue by ajplopez** [2sic/2sxc#2084]
**I'm submitting a ...** 
[x] bug report => search github for a similar issue before submitting 

**...about**   
[x] edit experience / UI

**Current behavior**

Using this method, <%=ToSic.Dnn.Factory.CmsBlock(XY, XZ).Render()%>, to load a Content module into the DNN skin, it works but when trying to edit content on the module (either editing content or reordering items), an alert is shown and an error logged in the Console of the browser inspector tool.

![Screen Shot 2020-05-20 at 8 56 49 AM](https://user-images.githubusercontent.com/8751936/82468720-00e54580-9a78-11ea-8244-8eb52044afbc.png)

![Screen Shot 2020-05-20 at 9 00 56 AM](https://user-images.githubusercontent.com/8751936/82469120-794c0680-9a78-11ea-824e-a63be8ffd215.png)

The changes are saved but when you edit items, the page throws an alert error and you have to refresh the page to see the changes.

**Expected behavior**
Content editors should be able to edit the content module without errors.

**Minimal reproduction of the problem with instructions**

1. Create a content module on the homepage
2. Edit your .ascx skin file to include the content module using: <%=ToSic.Dnn.Factory.CmsBlock(XY, XZ).Render()%>
3. Once you see the module loading in the skin via that method, edit an item. Or try to reorder items.
4. After you click save, you will see an alert in your browser that says "Had an error talking to the server (status 400). Message: Specific module instance does not exist."

![Screen Shot 2020-05-20 at 8 56 49 AM](https://user-images.githubusercontent.com/8751936/82468720-00e54580-9a78-11ea-8244-8eb52044afbc.png)

**What is the motivation / use case for changing the behavior?**
It's a bug. I can see this method of inserting modules directly into the skins very useful for banner notifications, header content like phone numbers, footer links, menus, social media icons, etc.

**Please tell us about your environment:**

Any

* **2sxc version(s):** 11.0.1 
* **Browser:** [all]
* **DNN:** [all] 
* **Language:** [any/all | English | German | other ]

**Anything you would like to add**

In reference to this solution:
https://stackoverflow.com/questions/61217797/is-it-possible-to-load-a-specific-2sxc-module-in-a-dnn-skin

**Comment by iJungleboy** [comment]
We'll look at this some time, but doesn't have a priority. Editing is something we didn't really focus on in this scenario, because editing on each page is usually not something you want.

**Comment by iJungleboy** [comment]
The idea is that the content is created in a real page, and the skin-include just distributes it to each page.

**Comment by ajplopez** [comment]
Thanks Daniel. I totally understand that it's not a priority. 

For what it's worth, I strongly believe this method to be superior to Adding Existing Module to each page or using the toggle for displaying module on all pages. Because from the UX perspective of my clients / content editors, changing content in the footer should be a simple point and click, regardless of what page they're on. Adding the module via the skin prevents them from deleting the module by accident, or moving it, or changing its permissions which is ideal. From the developer side, it's super simple and fast to implement and easy to swap out with another module if need be.

While I won't be using this method a lot on a single website, I definitely will be using it on every single website I create going forward because it's so useful.

As always, thanks for the amazing work on this module :D

**Comment by iJungleboy** [comment]
BTW: we ran into a similar issue recently, but it turned out the reason was that the "original" module was deleted from the page it was on. 
Could this also be your issue? Or that the page is not accessible by admins?

**Comment by ajplopez** [comment]
In my setup, the original module was on a page that is visible by all users. In fact, when I first tested this feature, I did it backwards. My original module was on the homepage and visible by all users. Then I had a skin that loaded the module from the homepage. And that skin was applied to a page that was not visible by all users - only admins. Then I tested editing on this hidden page and that's where I got the error.

However.... I'm testing a completely different website running 2sxc 11.2 and I'm not running into this problem anymore.
